src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -83,25 +83,20 @@ export const TranscriptsListItemInfo = (
     objectId: props.gene.unversioned_stable_id
   });
 
-  const getLinkToProteinView = (transcriptId: string) => {
-    if (transcript.so_term !== 'protein_coding') {
-      // TODO: Use Protein ID
-      return transcriptId;
-    }
+  const getLinkToProteinView = (stableId: string) => {
```

Comment:
`stableId` => `proteinStableId` (just to avoid confusion)?

Reply:
That's done